packages/account/src/Sections/Security/Passkeys/passkeys-configs.tsx

```diff
@@ -113,7 +113,22 @@ type TGetModalContent = {
     is_passkey_registration_started: boolean;
 };
 
+export const not_supported_error_name = 'NotSupportedError';
```

Comment:
shouldnt this be in uppercases `NON_SUPPORTED_ERROR_NAMES` as per accounts package naming convention?